test/food-test.js

```diff
@@ -2,6 +2,9 @@ pry = require("pryjs");
 const Food = require("../lib/food");
 const makeFoods = require("../lib/foods").makeFoods;
 const assert = require("chai").assert;
+require('jsdom-global')()
+global.$ = require('jquery');
+
 
```

Comment:
Added these lines in this test and the other in an attempt to get mocha to recognize jquery but it's not working

Reply:
@laurenfazah @case-eee @kat3kasper Any ideas on how to get mocha to recognize our jQuery? I added jsdom and jsdom-global to package.json and these lines to our tests, but it's still throwing an error because Mocha doesn't recognize $ as jQuery like the browser does.

Reply:
Since we weren't working together today, I'm not sure what @katiekeel has and hasn't tried, but  I have had similar problems on my end related , for although i've managed to get my tests working they depend on requiring (@laurenfazah) the code you gave me, and that breaks the browser rendering. @kat3kasper has offered to help us out tomorrow. Is there anything specific you'd like us to try before then?

Reply:
I was talking to Daniel about this earlier today and gave him pointers to jsdom. It sounds like you two have been trying to debug this for a while, so at this point, if it's still not working, I'd recommend moving on with other features. I believe @kat3kasper offered some time to help tomorrow, so maybe return back to this then?